MidiPlayer/nbproject/licenseheader.txt

```diff
@@ -1,44 +0,0 @@
-<#if licenseFirst??>
```

Comment:
Please restore this file and put it in the root of the project (same place as the pom.xml).

Reply:
Restored